### PR TITLE
[python] in prologue, read robot model from ros parameter.

### PR DESCRIPTION
**Pull request by florent-lamiraux**
- build reduced model by detecting and removing mimic joints,
  - checked by comparing Jcom on initial configuration for both methods,
    difference is below 1e-17.

Requires https://gepgitlab.laas.fr/pyrene-dev/talos_data/merge_requests/8 and https://github.com/stack-of-tasks/talos_metapkg_ros_control_sot/pull/5.

### CODE REVIEW

**Comment by florent-lamiraux** [comment]
Successfully checked through geometric_simu.

**Comment by olivier-stasse** [comment]
Thanks for the nice contribution.

File: src/dynamic_graph/sot/talos/talos.py [hunk 1]
```diff
@@ -46,6 +48,8 @@ class Talos(AbstractHumanoidRobot):
                                (0.,1.,0.,0.),
                                (0.,0.,1.,-0.107),
                                (0.,0.,0.,1.))
+    defaultFilename = "talos_reduced_v2.urdf"
```
Comment (jmirabel) [inline src/dynamic_graph/sot/talos/talos.py]: Didn't you want to point to *pyrene.urdf* ?

File: src/dynamic_graph/sot/pyrene/prologue.py [hunk 2]
```diff
@@ -17,6 +17,11 @@
 
 from dynamic_graph.entity import PyEntityFactoryClass
 from dynamic_graph.sot.pyrene.robot import Robot
+# sys.argv is not defined when running the remove interpreter, but it is
+# required by rospy
+import sys
+if not hasattr (sys, 'argv'):
+    sys.argv = "dynamic_graph"
```
Comment (gabrielebndn) [inline src/dynamic_graph/sot/pyrene/prologue.py]: Normally, `sys.argv` is a list, where element 0 is the name of the python main file itself. So, I think `sys.argv = ["dynamic_graph"]` would be better

File: src/dynamic_graph/sot/talos/talos.py (hunk 1 above)
Comment (florent-lamiraux) [reply src/dynamic_graph/sot/talos/talos.py]: You are right, I will push the modification.

File: src/dynamic_graph/sot/pyrene/prologue.py (hunk 2 above)
Comment (florent-lamiraux) [reply src/dynamic_graph/sot/pyrene/prologue.py]: You are right, I will push the modification.

File: src/dynamic_graph/sot/talos/talos.py (hunk 1 above)
Comment (florent-lamiraux) [reply src/dynamic_graph/sot/talos/talos.py]: In fact no. A reduced model is expected here. I do not want to break existing application, but we need to read the full model from file in the future.

**Pull request merged**